src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.2.0-preview",
+  "version": "0.2.1-preview",
```

Comment:
Almost.  In AssemblyInfo.cs, the AssemblyVersion should be 0.2.0.0 as you have specified, the AssemblyFileVersion should match the package version in packjage.json (0.2.1.0)